[Keyboard] Add Kudox Full (#15256)
qmk/qmk_firmware: keyboards/kudox_full/readme.md

# The Kudox Full keyboard

![Kudox logo](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox.png)

![Kudox Full image2](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-full-image2.jpg) | ![Kudox Full image1](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-full-image1.jpg)
:----:|:-----:
 front | angle

> - Keyboard Maintainer: [Kumao Kobo](https://github.com/kumaokobo)  
> - Hardware Supported: Kudox Full PCB rev1.0 w/ Pro Micro  
> - Assembly guide: [Kudox Full/README](https://github.com/kumaokobo/kudox-keyboard/blob/master/kudox-full/README.md)

Reviewer: ```suggestion
- Keyboard Maintainer: [Kumao Kobo](https://github.com/kumaokobo)
- Hardware Supported: Kudox Full PCB rev1.0 w/ Pro Micro
- Assembly guide: [Kudox Full/README](https://github.com/kumaokobo/kudox-keyboard/blob/master/kudox-full/README.md)
```

> Make example for this keyboard (after setting up your build environment):

Reviewer 2: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

    make kudox_full/rev1:default

> 
> Example of flashing this keyboard:

Reviewer: ```suggestion
Flashing example for this keyboard:
```

# The Kudox Full keyboard

![Kudox logo](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox.png)

![Kudox Full image2](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-full-image2.jpg) | ![Kudox Full image1](https://raw.githubusercontent.com/kumaokobo/kudox-keyboard/master/img/kudox-full-image1.jpg)
:----:|:-----:
 front | angle

- Keyboard Maintainer: [Kumao Kobo](https://github.com/kumaokobo)  
- Hardware Supported: Kudox Full PCB rev1.0 w/ Pro Micro  
- Assembly guide: [Kudox Full/README](https://github.com/kumaokobo/kudox-keyboard/blob/master/kudox-full/README.md)

Make example for this keyboard (after setting up your build environment):

> ```sh
> make kudox_full/rev1:default
> ```
> 
> 
> Example of flashing this keyboard:
> 
> ```sh
> make kudox_full/rev1:default:flash
> ```

Reviewer: ```suggestion
    make kudox_full/rev1:default

Flashing example for this keyboard:

    make kudox_full/rev1:default:flash
```

# The Kudox Full keyboard

> <p align="center">

Reviewer 2: please remove all html code from the readme and use markdown, instead.
